lib/internal/per_context/primordials.js

```diff
@@ -420,6 +420,12 @@ primordials.SafeSet = makeSafe(
   Set,
   class SafeSet extends Set {
     constructor(i) { super(i); } // eslint-disable-line no-useless-constructor
+
+    toUnsafeSet() {
+      const set = new Set();
+      this.forEach((value) => primordials.SetPrototypeAdd(set, value));
+      return set;
```

Comment:
Can we write tests that validates no user-code would be run? I think this can be simplified to
```suggestion
      return new Set(this);
```

Reply:
Sure! I couldn't find anywhere `SafeSet` is already tested, so i'm not sure where to add them - any suggestions?

I guess this is a valid simplification since it has a safe `[Symbol.iterator]` on it, in which case you're right, it's probably not needed. I was assuming add-at-construction-time wasn't an option.